examples/examples.py

```diff
@@ -58,8 +58,8 @@
     - local_db: without database and grafana
     - timescale: with database and grafana (note: you need docker installed)
     """
-    data_format = "timescale"  # "local_db" or "timescale"
-    example = "learning_small"
+    data_format = "local_db"  # "local_db" or "timescale"
+    example = "small"
```

Comment:
Is this change inteded?

Reply:
yes, since we don't expected people to already have docker running when using this small example.